XMonad/Actions/TreeSelect.hs

```diff
@@ -522,18 +523,22 @@ moveWith f = do
 -- | wait for keys and run navigation
 navigate :: TreeSelect a (Maybe a)
 navigate = gets tss_display >>= \d -> join . liftIO . allocaXEvent $ \e -> do
-    maskEvent d (exposureMask .|. keyPressMask .|. buttonReleaseMask) e
+    -- See XMonad.Prompt Note [Allow ButtonEvents]
+    maskEvent d (exposureMask .|. keyPressMask .|. buttonReleaseMask .|. buttonPressMask) e
+    allowEvents d replayPointer currentTime
```

Comment:
You forgot this here.

Reply:
Ah good catch.  I moved the comment concerning the note down; or do you think it's necessary to have it up here as well?

Reply:
It's not necessary.